### ISSUE TITLE
a infinite loop happens

### ISSUE THREAD

**Issue by jras13** [riderkick/FMD#1059]
i updated to latest FMD, and loop happened.. like.. keep repeating load downloaded list
repeating load favourites
repeating load failed download
all files on FMD/works keep loading without end.. and make app crash.. what happened?

**Comment by kmvi** [comment]
yea, something strange happens...
![untitled](https://user-images.githubusercontent.com/6571513/36679832-42235c8c-1b25-11e8-8330-328dc10e4b9a.png)

**Comment by jras13** [comment]
![image](https://user-images.githubusercontent.com/20046326/36679968-1d5a433a-1b47-11e8-88ec-b34e931fc1b6.png)
ugh.. it keep going and crash

**Comment by kmvi** [comment]
it seems that the release 0.9.150.0 no longer has this error, try to update manually
https://github.com/riderkick/FMD/releases/tag/0.9.150.0

**Comment by jras13** [comment]
![image](https://user-images.githubusercontent.com/20046326/36680573-b3d9f7be-1b48-11e8-8b5b-9e7d3f1bece8.png)
solved
sorry, im in panic mode ^^ and thanks

**Comment by riderkick** [comment]
That's the bug that I'm referring yesterday. When adding duplicate website option (fail to set the pointer). In this case mangadex. It will fail the startup timer and causing it take longer so the timer have a time to loop due to the interval is to low. Fixed with commit that destroy the timer immedietly. That's why I push the mangadex module update after the release.

**Comment by molitar** [comment]
Thanks for the ix.. Too bad I lost my entire manga list wen this happened as manual install overwrote it all now I have to wait forever again to update every single site.

**Comment by LakotaGray** [comment]
When I opened FMD it first sought to update a list of 7 modules, as well as notified of the newest version update. I update modules and allowed restart, and the constant module adding ensued. Closed and opened several times but same issue. It wouldn't allow an update as it was glitching before getting there. Closed FMD, I manually deleted the newest modules and then reopened. Notifications for both new mods and version popped up again, this time updating FMD before modules. Restarted program, then allowed modules to update and it worked out fine, or at least so far it seems to be.
